cpp/include/cugraph/algorithms.hpp

```diff
@@ -2349,6 +2349,25 @@ rmm::device_uvector<vertex_t> maximal_independent_set(
   graph_view_t<vertex_t, edge_t, false, multi_gpu> const& graph_view,
   raft::random::RngState& rng_state);
 
+/*
+ * @brief Find a Minimal Graph Coloring
+ *
```

Comment:
I feel like we need a bit more description of what this algorithm is doing.

Reply:
Added.